Resources/public/js/views/fields/ez-float-editview.js

```diff
@@ -13,7 +13,7 @@ YUI.add('ez-float-editview', function (Y) {
 
     var L = Y.Lang,
         FIELDTYPE_IDENTIFIER = 'ezfloat',
-        FLOAT_PATTERN = "\\-?\\d*\\.?\\d+"; // WARNING: each backslash is doubled, because it is escaped on output otherwise
```

Comment:
if I get the issue right, Firefox now complains when if some characters are escaped while that's not necessary. As far as I can tell, here `-` does not need to be escaped so I guess using:
```js
FLOAT_PATTERN = "-?\\d*\\.?\\d+";
```
should fix the issue (and this would result in a much small patch)